release-notes/v5.6.0.md

```diff
@@ -23,11 +23,9 @@
 
 #### <sub><sup><a name="v560-note-4311" href="#v560-note-4311">:link:</a></sup></sub> feature
 
-* Credentials fetched from a credential manager will now be automatically redacted from build output, thanks to a couple of PRs by @evanchaoli! #4311
+* Credentials fetched from a credential manager will now be automatically redacted from build output, thanks to a couple of PRs by @evanchaoli! #4311 #4398
 
-  Please don't rely on this functionality to keep your secrets safe: you should continue to prevent accidental credential leakage, and only treat this as a safety net.
-
-  > NOTE: In its current form, credentials that get 'split' into multiple `write()` calls will not be redacted. This may happen for large credentials, or if you're just unlucky. Work to improve this is in-progress: #4398
+  This feature is currently opt-in. To learn how to enable it, check out the [docs](https://concourse-ci.org/creds-redacting.html).
```

Comment:
`creds-redacting.html` will be submitted to `concourse/docs` repo as a separate PR right

Reply:
https://github.com/concourse/docs/pull/255